include/mgclient.hpp

```diff
@@ -189,6 +195,19 @@ inline std::optional<std::vector<Value>> Client::FetchOne() {
   return values;
 }
 
+inline void Client::DiscardAll() {
+  while (FetchOne())
```

Comment:
Writing this smarter would also be a cool onboarding/nice to have feature :thinking: 
Like buffering data and calling PULL/DISCARD messages when necessary.